tools/generation/lib/template.py

```diff
@@ -154,7 +154,7 @@ def _frontmatter(self, case_filename, case_values):
         features += self.attribs['meta'].get('features', [])
         features = list(OrderedDict.fromkeys(features))
         if len(features):
-            lines += ['features: ' + yaml.dump(features).strip()]
+            lines += ['features: ' + re.sub('\n|\r', '', yaml.dump(features).strip())]
```

Comment:
My earlier suggestion would be `sub('\n\s*', ' ')`.

Rationale:

- Lone CR line endings are not really a thing although CRLF is. That is, I don’t think you want to match lone `\r`. 
- `.editorconfig` already enforces LF line endings. https://github.com/tc39/test262/blob/ff8b10c448ab0b884deac4e6e5818ed84ab8ecd9/.editorconfig#L6